quirks/password-rules.json

```diff
@@ -47,6 +47,9 @@
     "claimlookup.com": {
         "password-rules": "minlength: 8; maxlength: 16; required: lower; required: upper; required: digit; required: [@#$%^&+=!];"
     },
+    "claro.com.br": {
+        "password-rules": "minlength: 8; required: lower; allowed: [abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789!@#$%&*_-+=<>];"
```

Comment:
This is equivalent to `minlength: 8; required: lower; allowed: upper, digit, [!@#$%&*_-+=<>];`.

Is it true that they only require a lower, and not upper or digit?

Reply:
yes :(